src/coprocessor/executor/aggregation.rs

```diff
@@ -21,12 +21,12 @@ use tipb::executor::Aggregation;
 use tipb::expression::Expr;
 
 use util::collections::{HashMap, HashMapEntry as Entry};
-use util::codec::datum::{self, DatumEncoder, approximate_size};
-use util::codec::table::RowColsDict;
-use util::xeval::{Evaluator, EvalContext};
 
 use super::super::Result;
 use super::{Executor, Row, ExprColumnRefVisitor};
```

Comment:
I'm not sure whether there is rule defined in rust for organizing these import paths.

This order of putting further paths ahead like fine to me, as long as it's used consistently:
```
use super::super::XXX;
use super::XXX::xxx;
use super::xxx;
```